### PR TITLE
Fix/cxspa 6450 avoid too many calls

### PR DESCRIPTION
**Pull request by ChristophHi**

### CODE REVIEW

File: feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.ts [hunk 1]
```diff
@@ -144,21 +145,14 @@ export class ConfiguratorAddToCartButtonComponent implements OnInit, OnDestroy {
...
-            configuration
-          )
+          this.configuratorCommonsService.getConfiguration(routerData.owner)
         ),
         filter((configuration) => configuration.overview != null),
         take(1),
         delay(0) //we need to consider the re-rendering of the page
       )
       .subscribe(() => {
+        this.focusService.clear();
```
Comment (Uli-Tiger) [inline feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.ts:155]: if i remove this line - the tests still pass, so i assume an additional test and/or an additional assert is missing.
the same issue holds true for the tab-bar component

File: feature-libs/product-configurator/rulebased/core/facade/configurator-commons.service.ts [hunk 2]
```diff
@@ -183,11 +185,20 @@ export class ConfiguratorCommonsService {
...
       select(
-        ConfiguratorSelectors.getConfigurationFactory(configuration.owner.key)
+        ConfiguratorSelectors.getConfigurationProcessLoaderStateFactory(
+          configuration.owner.key
+        )
       ),
-      filter((config) => this.configuratorUtils.isConfigurationCreated(config)),
-      tap((configurationState) => {
-        if (!this.hasConfigurationOverview(configurationState)) {
+      filter((configurationState) => configurationState.loading === false),
```
Comment (Uli-Tiger) [inline feature-libs/product-configurator/rulebased/core/facade/configurator-commons.service.ts]: if i remove this line the tests still pass

**Review by Uli-Tiger — CHANGES_REQUESTED** [review]
@steinsebastian : could you plz, add test for your code changes: https://github.com/SAP/spartacus/pull/18601/commits/9eccbef35e14f86bec6c990a229cc72a4a7b65cc
@ChristophHi : would be great to have a test that fails if the additional filter in the configurator commons service is removed

File: feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.ts (hunk 1 above)
Comment (steinsebastian) [reply feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.ts:155]: Done.

**Review by Uli-Tiger — APPROVED** [review]

**Pull request merged**